shopify_integration/hook_events/connected_app.py

```diff
@@ -0,0 +1,16 @@
+from typing import TYPE_CHECKING
+from urllib.parse import urljoin
+
+import frappe
+
+if TYPE_CHECKING:
+	from frappe.integrations.doctype.connected_app.connected_app import ConnectedApp
+
+
+def validate_redirect_uri(connected_app: "ConnectedApp", method: str):
+	if "shopify" in connected_app.provider_name.lower():
```

Comment:
If you think about it now, do you think there's a way to do this in a better way? This is pretty hacky.